assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -66,12 +66,12 @@ const Cart = () => {
 	/**
 	 * Given an API response with cart totals, generates an array of rows to display in the Cart block.
 	 *
-	 * @param {Object} cartTotals - Cart totals data as provided by the API.
+	 * @param {Object} fetchedCartTotals - Cart totals data as provided by the API.
```

Comment:
This docblock is out of date, can delete `fetchedCartTotals`, if you keep `getTotalRowsConfig` inside the component :)